AirCasting/AuthViews/CreateAccountView.swift

```diff
@@ -24,12 +24,22 @@ struct CreateAccountView: View {
     @State private var isUsernameBlank = false
     @State private var presentedError: AuthorizationError?
     @State private var alert: AlertInfo?
-    
-    init(completion: @escaping () -> Void) {
+    @State private var isActive: Bool
```

Comment:
I'm not sure it's the right variable name. What exactly is active? I this a better name would be: `isLoading`, because we are setting it to true when data is loading, right? Also, I think it could be set to false here instead of a value being passed to the initializer like with the rest of @State variables.